frontend/src/component/feature/CreateFeature/useFlagLimits.test.ts

```diff
@@ -0,0 +1,71 @@
+import { renderHook } from '@testing-library/react';
+import { useFlagLimits } from './CreateFeature';
+import { vi } from 'vitest';
+
+vi.mock('hooks/useUiFlag', async (importOriginal) => {
+    const actual = await importOriginal();
+    return {
+        ...(actual as {}),
+        useUiFlag: (flag: string) => flag === 'resourceLimits',
+    };
+});
+
+test('if both global and project-level limits are reached, then the error message shows the message for instance-wide limits', () => {
+    const { result } = renderHook(() =>
```

Comment:
it's a mater of personal preferences but I usually create a hand written test component for my hooks to test them. However this one is also fine.

Reply:
Right. I'm pretty sure we do this a few other places, so I'm inclined to keep it. But if you write hand written test components for it, you probably have a good reason for it. Why do you prefer those components?

Reply:
Mostly to show hook usage in context of some component. I know that renderHook is like a simulation of an anonymous component but I still prefer the explicitness of my own components. But as you said, we have both examples in the code so feel free to ignore my comment.